tensorflow_blade/src/custom_ops/non_max_suppression/gpu_non_max_suppression_op.cu.cc

```diff
@@ -0,0 +1,614 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* ==================================================
+ * None Maximum Suppression Operation - GPU Version
+ *
+ * Author:
+ *   Yifan Lu (evanlu.lyf@alibaba-inc.com)
+ *
+ ================================================== */
+
+#ifndef __NONE_MAXIMUM_SUPPRESSION_OP_GPU_DEVICE__
+#define __NONE_MAXIMUM_SUPPRESSION_OP_GPU_DEVICE__
+
```

Comment:
Is it necessary to release these custom ops to Github?
I tend to keep them in the internal repository since: (1) the architecture is confusing to external readers; (2) I don't see strong necessity to do this.